### PR TITLE
Build the "Awesome books" website using classes

### PR DESCRIPTION
**Pull request by miltonHenschel**
- [ ] Use JavaScript classes instead of objects and arrays
- [ ] Create class methods to add and remove books
- [ ] Add CSS styles to the application to make it match the Basic UI wireframe

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -7,133 +7,100 @@ const titleInput = document.querySelector('.title-book-add');
...
-  const newBook = new Book(newTitle, newAuthor);
-  const ui = new UI();
-  ui.deleteBook(newBook);
+  if (e.target.className === 'btn-remove') {
+    const { id } = e.target;
+    let books = LocalStore.getBooks();
+    books = books.filter((bk) => JSON.stringify(bk.id) !== id);
+    localStorage.setItem('local', JSON.stringify(books)); // LOCAL STORAGE
+    UI.deleteBookFromList(e.target);
+  }
```
Comment (OyePriscilla) [inline main.js:103]: - [ ] Please kindly take note that your delete function is not working perfectly as expected, whenever a book is removed form the UI I noticed that your localstorage still retain it and this makes it to persist on the UI. ℹ️

File: main.js [hunk 2]
```diff
@@ -7,133 +7,100 @@ const titleInput = document.querySelector('.title-book-add');
...
+}
+
+// ADD BOOKS
 
 btnAdd.addEventListener('click', (e) => {
   e.preventDefault();
+  const books = LocalStore.getBooks();
   const newTitle = titleInput.value;
   const newAuthor = authorInput.value;
```
Comment (OyePriscilla) [inline main.js:79]: - [ ] Please kindly ensure that empty value cannot be added to the list. 💁‍♂️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @otmanTR   😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️
- [ ] Please ensure you ignore node_modules folder.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (miltonHenschel) [reply main.js:103]: Thanks for the input but the remove button works fine. You just need to reload the page. Also, it works well on my side but after deploying the site, errors shows on G. Chrome but not on M. Edge. Please could you see into that??! Thanks. :pray:

File: main.js (hunk 2 above)
Comment (miltonHenschel) [reply main.js:79]: Thanks @OyePriscilla, I'll do just that.

**Review by frank1738 — APPROVED** [review]
Hi @miltonHenschel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by otmanTR** [comment]
Hello  ***@***.***;*

I think you sent this email wrong person. Could you check pull request, it
isn't belong to me my friend.
Greetings

**Comment by otmanTR** [comment]
Hah I see now:) Thank you to remember me. I hope your week was well. See
you around my friend.
Greetings

**Pull request merged**